mouhamadouBarry/assignments/reverseLinked/lc206/lc206.js

```diff
@@ -1,16 +1,41 @@
 // Problem Statement #
 
 // Given the head of a Singly LinkedList, reverse the LinkedList. Write a function to return the new head of the reversed LinkedList.
+/*
 
+LOOK THE METHOD THAT A LINKED LIST HAS
+->they do not have a length method
+
+INPUT: A singly Linkedlist
+OUTPUT: Return a reversed the Linkedlist
+
+LOGIC:
+
+  let previous = null
+  let current = head
+
+  while(head.next){
+
+    let next = current.next
+    current.next = previous
+    let previous = current
+    current = next
+
+  }
+  return previous

```

Comment:
This is not really psuedocode, it's just code. Psuedocode helps you understand what your code will do to complete the prompt. This approach would benefit from being written out in sentences to show your understanding of the process and the prompt.